certified-connectors/WQRM Risk Forecast Services/apiDefinition.swagger.json

```diff
@@ -0,0 +1,326 @@
+                "Name": {
+                  "type": "string",
+                  "description": "Name"
+                },
+                "TenantFirstName": {
+                  "type": "string",
+                  "description": "TenantFirstName"
+                },
+                "TenantLastName": {
+                  "type": "string",
+                  "description": "TenantLastName"
+                },
+                "CompanyName": {
+                  "type": "string",
+                  "description": "CompanyName"
+                },
+                "Email": {
+                  "type": "string",
+                  "description": "Email"
+                }
+              }
+            }
+          }
+        ]
+      }
+    },
+    "/play/158ed27b-9e89-45d2-a216-617d0b2d4355": {
+      "post": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
+          }
+        },
+        "summary": "Bank Report",
+        "description": "Process bank report and email communication.",
+        "operationId": "ReportManagementBanks",
+        "parameters": [
+          {
+            "name": "tenantId",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "Bank ID",
+            "x-ms-summary": "Bank ID number."
+          },
+          {
+            "name": "Content-Type",
+            "in": "header",
+            "required": false,
+            "type": "string",
+            "description": "Content Type",
+            "x-ms-summary": "Content type number."
+          },
+          {
+            "name": "Accept",
+            "in": "header",
+            "required": false,
+            "type": "string",
+            "description": "Accept",
+            "x-ms-summary": "Accept number."
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": false,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "Name": {
+                  "type": "string",
+                  "description": "Name"
+                },
+                "TenantFirstName": {
+                  "type": "string",
+                  "description": "TenantFirstName"
+                },
+                "TenantLastName": {
+                  "type": "string",
+                  "description": "TenantLastName"
+                },
+                "CompanyName": {
+                  "type": "string",
+                  "description": "CompanyName"
+                },
+                "Email": {
+                  "type": "string",
+                  "description": "Email"
+                }
+              }
+            }
+          }
+        ]
+      }
+    },
+    "/play/b60262a8-7cf2-4526-8e78-c7fc7bd21ae9": {
+      "post": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
```

Comment:
@Chuck0007 empty schema is not allowed. please remove or define a proper schema.